core/email.py

```diff
@@ -310,9 +311,13 @@ def send_denied_resource_email(user, request, reason):
         "request": request,
         "reason": reason
     }
-    body = render_to_string("core/email/resource_request_denied.html",
-                            context=Context(context))
-    return email_from_admin(user, subject, body)
+    from_name, from_email = admin_address()
+    user_email = lookupEmail(user.username)
+    recipients = [email_address_str(user.username, user_email)]
+    sender = email_address_str(from_name, from_email)
+
+    return send_email_template(subject, template, recipients, sender,
+                               context=context, cc=[sender])
```

Comment:
This code is copied from [above](https://github.com/iPlantCollaborativeOpenSource/atmosphere/pull/167/commits/5b3a415328a42bf4a2a87783e4943133e3f2fe34#diff-8d164c7b0e814633663ff40f14c113aeL275)